test/common/libs/taskDefaults.test.js

```diff
@@ -1,8 +1,9 @@
 import taskDefaults from '../../../website/common/script/libs/taskDefaults';
+import { generateUser } from '../../helpers/common.helper';
 
 describe('taskDefaults', () => {
```

Comment:
can you add a new test for the new functionalities?

Reply:
Happy to add this test. The current tests don't work, I think due to my timezone (UK), so I may have to fix these first.